apps/judicial-system/web/src/utils/hooks/useSections/index.ts

```diff
@@ -408,6 +408,7 @@ const useSections = (
       isActive:
         isProsecutionUser(user) &&
         isIndictmentCase(type) &&
+        state !== CaseState.RECEIVED &&
```

Comment:
Consider also checking that `state !== CaseState.MAIN_HEARING` as that state can also occur during the court proceedings.